README.md

```diff
@@ -46,10 +46,10 @@ Verified on :
 
 	_Same as in previous step, use `~/.profile` on Ubuntu, `~/.zshrc` for Zsh._
 
-    **Fish note**: Instead, copy `~/.jenv/fish/jenv.fish` to `~/.conf/fish/function/jenv.fish`. If you don't have the `export` function, also copy `export.fish`
+    **Fish note**: Instead, copy `jenv.fish` to yor fish functions folder. If you don't have the `export` function, also copy `export.fish`
     ~~~ sh
-        cp ~/.jenv/fish/jenv.fish ~/.conf/fish/function/jenv.fish
-        cp ~/.jenv/fish/export.fish ~/.conf/fish/function/export.fish
+        cp /usr/local/Cellar/jenv/0.4.4/libexec/fish/jenv.fish ~/.config/fish/functions/jenv.fish
+        cp /usr/local/Cellar/jenv/0.4.4/libexec/fish/export.fish ~/.config/fish/functions/export.fish
```

Comment:
```suggestion
        cp ~/.jenv/fish/export.fish ~/.config/fish/functions/export.fish
```

Reply:
Step 1 considers that the user is installing jenv through `git clone` and it is located at `~/.jenv`. I would keep that and update both `.config` and `functions`